docs/source/daml-script/index.rst

```diff
@@ -9,7 +9,7 @@ DAML Script
 
    daml-script-docs
 
-DAML scenarios provide a simple API for testing DAML models
+DAML scenarios provide a simple way for testing DAML models
```

Comment:
What I was calling API here is things like `submit`, i.e., the scenario specific API provided by `daml-stdlib` not the gRPC API of the scenario service. But I do agree that your formulation is better :+1: